src/components/add-cash/AddCashStatus.js

```diff
@@ -100,15 +106,25 @@ const AddCashFailed = ({ error, resetAddCashForm }) => {
           You were not charged.
         </StatusMessageText>
       )}
-      {tryAgain ? (
+      {orderId && (
+        <OrderIdText>
+          <Br />
+          Order ID: {orderId}
+        </OrderIdText>
+      )}
+      <Row>
         <SupportButton
           label="Try again"
           marginTop={24}
           onPress={resetAddCashForm}
         />
-      ) : (
-        <NeedHelpButton marginTop={24} subject="Purchase Failed" />
-      )}
+
+        <NeedHelpButton
+          marginLeft={10}
+          marginTop={24}
+          subject={`Purchase Failed - Order ${orderId}`}
```

Comment:
Wanna check if is `orderId` is set before adding " - Order" to the string?

Reply:
woops I totally meant to do that